pkg/performanceprofile/controller/utils.go

```diff
@@ -0,0 +1,51 @@
+package controller
+
+import (
+	"context"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	olmoperators "github.com/operator-framework/api/pkg/operators/install"
+	olmv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+)
+
+func ListPerformanceOperatorCSVs(k8sclient client.Client, options []client.ListOption, paginationLimit uint64) ([]olmv1alpha1.ClusterServiceVersion, error) {
+	// Register OLM types to the client
+	olmoperators.Install(k8sclient.Scheme())
+
+	var performanceOperatorCSVs []olmv1alpha1.ClusterServiceVersion
+
+	csvs := &olmv1alpha1.ClusterServiceVersionList{}
+	initialOpts := append(options, client.Limit(paginationLimit))
+	opts := initialOpts
+
+	for {
+		if err := k8sclient.List(context.TODO(), csvs, opts...); err != nil {
+			if !errors.IsNotFound(err) {
+				return performanceOperatorCSVs, err
+			}
+		}
+		continueToken := csvs.GetContinue()
+
+		for i := range csvs.Items {
+			csv := &csvs.Items[i]
+			deploymentSpecs := csv.Spec.InstallStrategy.StrategySpec.DeploymentSpecs
+
+			for _, deployment := range deploymentSpecs {
+				if deployment.Name == "performance-operator" {
```

Comment:
can we detect (maybe pass as argument) the expected name from a reliable source? IOW, is it safe to hardcode here?

Reply:
yeah, sure we can ... 
it was hardcoded in the original code and I forgot to change it.